src/dom.js

```diff
@@ -16,6 +16,60 @@
 
 
 /**
+ * Waits until the child element is constructed. Once the child is found, the
+ * callback is executed.
+ * @param {!Element} parent
+ * @param {function():boolean} checkFunc
+ * @param {function()} callback
+ */
+export function waitForChild(parent, checkFunc, callback) {
+  if (checkFunc()) {
```

Comment:
pass in parent to `checkFunc` to follow all other `predicate` type signatures. not useful in this case since you have a ref to `parent` but could see it being useful in other scenarios. (might also be useful to pass it into callback)